test/system/700-play.bats

```diff
@@ -733,3 +733,24 @@ spec:
     run_podman 125 kube play --authfile=$bogus - < $PODMAN_TMPDIR/test.yaml
     is "$output" "Error: checking authfile: stat $bogus: no such file or directory" "$command should fail with not such file"
 }
+
+@test "podman kube play with umask from containers.conf" {
+    skip_if_remote "remote does not support CONTAINERS_CONF*"
+    YAML=$PODMAN_TMPDIR/test.yaml
+
+    containersConf=$PODMAN_TMPDIR/containers.conf
+    touch $containersConf
+    cat >$containersConf <<EOF
+[containers]
+umask = "0077"
+EOF
+
+    ctr="ctr"
+    ctrInPod="ctr-pod-ctr"
+
+    run_podman create -t --restart never --name $ctr $IMAGE
+    run_podman kube generate -f $YAML $ctr
+    CONTAINERS_CONF_OVERRIDE="$containersConf" run_podman kube play $YAML
+    run_podman container inspect --format '{{ .Config.Umask }}' $ctrInPod
+    is "${output}" "0077"
+}
```

Comment:
system tests should always cleanup on their own so you need to rm the container and run something like podman kube down

Reply:
Fixed.